<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-ROFA72B" failures="1" tests="1" name="Default test" time="19.296" errors="0" timestamp="2023-09-12T07:09:21 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="17.857">
    <failure type="java.lang.AssertionError" message="User is not logged into the application...! expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: User is not logged into the application...! expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1067)
at org.testng.Assert.assertEquals(Assert.java:1031)
at com.test.stepDefinitions.LoginStepDef.validate_login(LoginStepDef.java:69)
at âœ½.Validate login(file:///C:/Users/MY%20PC/eclipse-workspace/CucumberTestAutomation/src/test/resources/features/LoginTest.feature:12)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Default test -->
